src/components/scrim/edit-scrim-form.tsx

```diff
@@ -205,11 +217,55 @@ export function EditScrimForm({
                 <FormDescription>
                   This is the team that will be associated with this scrim. You
                   can assign scrims to teams that you manage or own.
-                </FormDescription>
-                <FormMessage />
-              </FormItem>
-            )}
-          />
+              </FormDescription>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+        
+        <FormField
+          control={form.control}
+          name="date"
+          render={({ field }) => (
+            <FormItem className="flex flex-col">
+              <FormLabel>Scrim Date</FormLabel>
+              <Popover>
+                <PopoverTrigger asChild>
+                  <FormControl>
+                    <Button
+                      variant="outline"
+                      className={cn(
+                        "w-[240px] pl-3 text-left font-normal",
```

Comment:
Here the date field does not fit the width of the rest of the form fields. We can change the `w-[240p]` to `max-w-lg` to fix this issue.

![CleanShot 2024-06-12 at 14 40 09@2x](https://github.com/luxdotdev/parsertime/assets/55987186/683c6b07-6fd0-44b4-8f1c-026e97d3ef21)